SmartDeviceLink/SDLRegisterAppInterface.m

```diff
@@ -45,7 +45,7 @@ - (instancetype)initWithLifecycleConfiguration:(SDLLifecycleConfiguration *)life
                     shortAppName:lifecycleConfiguration.shortAppName
                          ttsName:lifecycleConfiguration.ttsName
                       vrSynonyms:lifecycleConfiguration.voiceRecognitionCommandNames
-       hmiDisplayLanguageDesired:lifecycleConfiguration.language
+       hmiDisplayLanguageDesired:lifecycleConfiguration.hmiLanguage
```

Comment:
This was not agreed upon, please revert.

Reply:
This has been reverted.